scalableminds/webknossos
# Enabled arbitrary mode in view only mode

This PR enables both arbitrary view modes in view only mode.

### URL of deployed dev instance (used for testing):
- [https://addarbitrarytoviewonlymode.webknossos.xyz/](https://addarbitrarytoviewonlymode.webknossos.xyz/)

### Steps to test:
- open a view only mode e.g. via the dataset gallery and try to open is in both arbitrary modes
- skeleton; volume and hybrid tracings should work normally

### Issues:
- fixes #3629

------
